script/glam/run_glam_sql

```diff
@@ -187,42 +187,59 @@ function run_desktop_sql {
 function run_glean_sql {
     local product=$1
     local start_stage=${START_STAGE:-0}
+    local reset=${RESET:-false}
+    local backfill_only=${BACKFILL_ONLY:-false}
+    local export_only=${EXPORT_ONLY:-false}
+
+    if [[ $backfill_only = true && $export_only = true ]]; then
+        echo "BACKFILL_ONLY and EXPORT_ONLY are mutually exclusive"
+        exit 1
+    fi
+    if [[ $reset = true && $export_only = true ]]; then
```

Comment:
I think it would be better to put this check before the `bq rm -r -f` so that if you wanted to just export but accidentally left reset true, you wouldn't delete everything.